torch/csrc/jit/serialization/inlined_callstack_serialization.cpp

```diff
@@ -0,0 +1,212 @@
+#include <torch/csrc/jit/api/compilation_unit.h>
```

Comment:
That should have been tested the unit tests added in this PR, no? Meaning the fact that module hiearchy and source debug info is tested should be sufficient? Or is there something that is missed. Of course will add similar test on cpp side, but I thought that should be sufficient.

Reply:
@iseeyuan You mean an actual unit test that calls say serialize and asserts against an IValue
I think that would also help as docs

Reply:
@kimishpatel From coverage point of view it's covered by other tests. I'm wondering for "unit" granularity it might be helpful to have just 
IValue1 -> serialize -> deserialize -> IValue2, and compare those two IValues are identical. As @raziel mentioned it also helps as a better documentation.

Reply:
@iseeyuan, ok that sounds good. Certainly it will be helpful to have that.

Reply:
Thanks for adding the test!